### PR TITLE
Readme suggestions

### PR DESCRIPTION
**Pull request by trymoto**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ example/
 src/
 .travis.yml
 README.md
+logo.png
```
Comment (iamolegga) [inline .npmignore]: logo will disappear at package page at npm.org

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img alt="NestJS-Pino logo" src="./logo.jpg" width="300" height="300" />
+  <img alt="NestJS-Pino logo" src="./logo.png"/>
```
Comment (iamolegga) [inline README.md]: why remove size?

Comment (trymoto) [reply README.md]: it'll just take available space

File: .npmignore (hunk 1 above)
Comment (trymoto) [reply .npmignore]: reverted

**Pull request merged**